dev/tidb-in-kubernetes/troubleshoot.md

```diff
@@ -152,7 +152,8 @@ When you find some network connection issues between Pods from the log or monito
 The Pending state of a Pod is usually caused by conditions of insufficient resources, such as:
 
 - The `StorageClass` of the PVC used by PD, TiKV, Monitor Pod does not exist or the PV is insufficient.
-- No nodes in the Kubernetes cluster can satisfy the CPU or memory applied by the Pod
+- No nodes in the Kubernetes cluster can satisfy the CPU or memory resources requested by the Pod
+- The replicas of TiKV or PD and the number of nodes in the cluster do not satisfy the high availability scheduling policy of tidb-scheduler
```

Comment:
```suggestion
- The number of TiKV or PD replicas and the number of nodes in the cluster do not satisfy the high availability scheduling policy of tidb-scheduler
```

Reply:
See https://github.com/pingcap/docs/pull/1628#discussion_r342348764. Lol

Reply:
@yikeke 😂Then there must be some issue with the Chinese description. @DanielZhangQD

Reply:
Hailong said that `replicas` also has the meaning of "number of desired pods". See [Kubernetes documentation](https://v1-12.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#deploymentspec-v1-apps). We discussed it and decided to apply Lilian's suggestion to avoid any kind of misunderstandings.